kubernetes/coredns.yaml.sed

```diff
@@ -51,6 +51,9 @@ data:
         log
         health
         kubernetes CLUSTER_DOMAIN SERVICE_CIDR POD_CIDR
+        prometheus {
```

Comment:
I added it because the Service below exports this port.

I'm not sure if I have to add parameters to bind to all interfaces. Default behavior is to bind to localhost only but I want to get this metrics via the exposed Service. I've noticed that kubeadm don't have it (https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/coredns.yaml.in#L63).